### PR TITLE
Issue #5 - Add Style

### LINKED ISSUE #5: Add Style
#### As a user I want to see a visually pleasing application.

__Story__:

Our web application is really starting taking shape. We have added a few
features that provide value to our customers, but some of our customers are
complaining that the website is a quite bland. It is time to spice it up by
applying a visually appealing, and easy to maintain user interface to our web
application.

__Acceptance criteria__:
- [x] The application has a header with navigation
- [x] Images are displayed in an aesthetically pleasing list of cards
- [x] Form errors are shown inline with the related input field
- [x] I know how to use Bootstrap components like buttons and navbars in my
  application.
- [x] I know the very basics of CSS and SCSS
- [x] My SCSS files are properly named and organized using BEM convention
- [x] My app has a "sticky" footer.
- [x] My app has a few customized styles using bootstrap variable overrides.
- [x] My app is responsive and is usable on a phone.
- [x] The active link in the navigation header is differentiated.
- [x] I use a bootstrap "container" to center my app contents (not necessarily "fluid").

__Reference material__:
- http://v4-alpha.getbootstrap.com/getting-started/introduction/
- https://github.com/plataformatec/simple_form
- http://getbem.com/naming/

__Dependencies__:
- Image Index

### PR DESCRIPTION
**Pull request by cvetanovskaa**
Closes #5

---

The following is a checklist to remind you of the steps to take when opening a
new PR. The horizontal line above and everything below can be removed from the
pull request description.

- [X] I have replaced `X` above with the associated issue number.
- [X] I have changed the issue's label to "pull request".
- [X] I have assigned the designated reviewer to this PR.
- [X] I have set the appropriate label on this PR.
- [X] I have written clear commit messages ([ref](http://chris.beams.io/posts/git-commit/)).

If needed, please review the document on the
[Image Sharing Project](https://sites.google.com/a/appfolio.com/eng/new-hire-resources/engineering-academy/ropes/image-sharing-project)
for information on how labels should be used.

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -13,3 +13,19 @@
...
+
+.footer_format {
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  background-color: #343a40;
+  color: white;
+}
+
+.fa.fa-navicon {
```
Comment (ramonrovirosa) [inline app/assets/stylesheets/application.css]: we should avoid using child/parent css styles and just do a style for 
```
.fa-navicon{
...
}
```
whenever possible

File: app/assets/stylesheets/images.css [hunk 2]
```diff
@@ -1,4 +1,21 @@
 .image_format{
   max-width:400px;
-  padding-top: 1em;
+  max-height: 250px;
+  padding: 1em 1em 1em 1em;
+}
+
+.cards{
```
Comment (ramonrovirosa) [inline app/assets/stylesheets/images.css]: for consistency you should add a space before the curly brace
e.g.
```.image_format {```

File: app/assets/stylesheets/images.css [hunk 3]
```diff
@@ -1,4 +1,21 @@
...
+  padding-left: 4em;
+}
+
+.image_card .card{
```
Comment (ramonrovirosa) [inline app/assets/stylesheets/images.css]: same as above, in BEM we would create a css class called something like ```.image_card--card``` and use that

File: app/assets/stylesheets/application.css (hunk 1 above)
Comment (cvetanovskaa) [reply app/assets/stylesheets/application.css]: Changed it

File: app/assets/stylesheets/images.css (hunk 2 above)
Comment (cvetanovskaa) [reply app/assets/stylesheets/images.css]: Removed it.

File: app/assets/stylesheets/images.css (hunk 3 above)
Comment (cvetanovskaa) [reply app/assets/stylesheets/images.css]: Used image_card--card class. Thank you!

**Comment by cvetanovskaa** [comment]
[Ropes Issue #5: Add Style](https://trello.com/c/4M4SyBEY/535-ropes-issue-5-add-style)

**Pull request merged**